tensorflow/tools/ci_build/Dockerfile.cpu

```diff
@@ -13,3 +13,11 @@ RUN /install/install_bazel.sh
 # Set up bazelrc.
 COPY install/.bazelrc /root/.bazelrc
 ENV BAZELRC /root/.bazelrc
+
+# Script for (optional) test on install
+COPY builds/test_on_install.sh /builds/test_on_install.sh
```

Comment:
You do not need to copy it in. You know the full path already. /tensorflow/tensorflow/tools/ci_build/builds/test_on_install.sh